FileManipulatorAbstractions/GlobalJsonModificationBuilder.cs

```diff
@@ -21,14 +22,14 @@ private GlobalJsonModificationBuilder(IImmutableList<MsBuildSdk> msBuildSdksToAd
 
         [Pure]
         public GlobalJsonModificationBuilder AddMsBuildSdk(MsBuildSdk sdk)
-            => ShallowClone(sdksToAdd: _msBuildSdksToAdd.Add(sdk));
+            => ShallowClone(sdksToAdd: Option.Some(_msBuildSdksToAdd.Add(sdk)));
```

Comment:
Use the implicit conversion here

Reply:
That does not work we work here on the `IImmutableList<T>`